core-java-modules/core-java-14/src/main/java/com/baeldung/java14/recordsCustomConstructors/StudentRecord.java

```diff
@@ -1,12 +1,11 @@
-package org.example;
-
-import java.util.Comparator;
-import java.util.List;
+package com.baeldung.java14.recordsCustomConstructors;
```

Comment:
Package names should be lowercase

Reply:
Oh, thanks for spotting it. I didn't check the package name.. fixed.